graph/types/blockchain.go

```diff
@@ -26,6 +27,25 @@ func UnmarshalAddress(v interface{}) (common.Address, error) {
 	}
 }
 
+func MarshalBytes(b []byte) graphql.Marshaler {
+	return graphql.WriterFunc(func(w io.Writer) {
+		out, _ := json.Marshal(b)
+		_, _ = io.WriteString(w, string(out))
+	})
+}
+
+func UnmarshalBytes(v interface{}) ([]byte, error) {
+	switch v := v.(type) {
+	case string:
+		out, _ := json.Marshal(v)
```

Comment:
This seems unlikely to work. `v` here is coming from naive JSON parsing: it ought to be the hex-encoded string, so something like `0xaf4b`. And then I think we're going to get the ASCII bytes for `"0xaf4b"`.